backoff.py

```diff
@@ -281,7 +292,15 @@ def retry(*args, **kwargs):
                                   'value': ret})
                         break
 
-                    seconds = next(wait) + jitter()
+                    try:
+                        if jitter is not None:
+                            seconds = jitter(next(wait))
```

Comment:
I think it's slightly better to have the next(wait) outside the try block here. The reason is in the case that the `wait` generator (which can be custom) throws a TypeError for some reason, the error might be masked.